src/scrapers/discount.ts

```diff
@@ -73,42 +78,47 @@ async function fetchAccountData(page: Page, options: ScraperOptions): Promise<Sc
       errorMessage: 'failed to get account data',
     };
   }
-  const accountNumber = accountInfo.UserAccountsData.DefaultAccountNumber;
 
   const defaultStartMoment = moment().subtract(1, 'years').add(2, 'day');
```

Comment:
Interesting, why's the previous developer used the `DefaultAccountNumber`, if the response includes all acounts?

I'm trying to say, we need someone with only one account to test your change, maybe for one account only, the `UserAccounts` is not exists or empty?

Reply:
Hi, just tested, works with one account